Error when generating a manual backup

Backups were working before the upgrade.

Version v8.0.4 - build 17196 (master)
Fresh upgrade, working well otherwise.

.Env backup settings:
MAIL_BACKUP_NOTIFICATION_DRIVER=null
MAIL_BACKUP_NOTIFICATION_ADDRESS=null
BACKUP_ENV=true
ALLOW_BACKUP_DELETE=true
ALLOW_DATA_PURGE=false


Error is:
"Error: Backup failed because: The dump process failed with a none successful exitcode. Exitcode ======== 2: Misuse of shell builtins Output ====== <no output> Error Output ============ mysqldump: [ERROR] unknown option '--skip-ssl'. . #0 C:\wamp64\www\snipe-it\vendor\spatie\db-dumper\src\DbDumper.php(200): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process)) #1 C:\wamp64\www\snipe-it\vendor\spatie\db-dumper\src\Databases\MySql.php(171): Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul(Object(Symfony\Component\Process\Process), 'C:\\wamp64\\www\\s "

![Image](https://github.com/user-attachments/assets/0cbcd0d5-bf31-4385-a8b6-96b39a61f3f3)



*Updated formatting

--- a/config/database.php
+++ b/config/database.php
@@ -8,6 +8,22 @@
  | be modified directly.
 */
 
+
+// This is used by the mysql dump options in spatie backup
+$dump_options = [
+    'dump_binary_path' => env('DB_DUMP_PATH', '/usr/local/bin'),  // only the path, so without 'mysqldump'
+    'use_single_transaction' => false,
+    'timeout' => 60 * 5, // 5 minute timeout
+    //'exclude_tables' => ['table1', 'table2'],
+    //'add_extra_option' => '--optionname=optionvalue',
+];
+
+// Some versions of mysql do not support the --skip-ssl option and will fail if it is
+if (env('DB_DUMP_SKIP_SSL') == 'true') {
+    $dump_options['skip_ssl'] = true;
+}
+
+
 return [
 
     /*
@@ -79,15 +95,7 @@ return [
             'strict'    => false,
             'engine'    => 'InnoDB',
             'unix_socket' => env('DB_SOCKET', ''),
-            'dump' => [
-                'dump_binary_path' => env('DB_DUMP_PATH', '/usr/local/bin'),  // only the path, so without 'mysqldump'
-                'skip_ssl' => env('DB_DUMP_SKIP_SSL', false),  // turn off SSL if not available
-                'use_single_transaction' => false,
-                'timeout' => 60 * 5, // 5 minute timeout
-                //'exclude_tables' => ['table1', 'table2'],
-                //'add_extra_option' => '--optionname=optionvalue',
-            ],
-
+            'dump' => $dump_options,
             'dump_command_timeout' => 60 * 5, // 5 minute timeout
             'dump_using_single_transaction' => true, // perform dump using a single transaction
             'options' => (env('DB_SSL')) ? ((env('DB_SSL_IS_PAAS')) ? [

